Doc/library/os.rst

```diff
@@ -4157,15 +4157,38 @@ written in Python, such as a mail server's external command delivery program.
 
    .. audit-event:: os.fork "" os.fork
 
+   .. warning::
+
+      If you use TLS sockets in an application calling ``fork()`` see
+      the warning in the :mod:`ssl` documentation.
```

Comment:
Are we able to link to a more specific target for the SSL warning than the module itself?
```suggestion
      If you use TLS sockets in an application calling ``fork()``, see
      the warning in the :mod:`ssl` documentation.
```